apps/web/app/(app)/onboarding/OnboardingEmailAssistant.tsx

```
@@ -0,0 +1,180 @@
+      )}
+    </div>
+  );
+}
+
+function EmailAssistantTestResults({
+  isLoading,
+  error,
+  data,
+}: {
+  isLoading: boolean;
+  error?: any;
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)